### PR TITLE
Avatar component

### PR DESCRIPTION
**Pull request by James300**
https://github.com/priceline/design-system/issues/645

Storybook example:
image

### CODE REVIEW

**Comment by James300** [comment]
[Avatar Component](https://trello.com/c/nbQaEioF/31-avatar-component)

File: packages/core/src/Avatar.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import styled from 'styled-components'
+import Flex from './Flex'
+import Heading from './Heading'
+import Text from './Text'
+import { User } from 'pcln-icons'
+
+const StyledImage = styled(Flex)`
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
```
Comment (unscsprt) [inline packages/core/src/Avatar.js]: Would it make sense for the size to be configurable? One use case might be a small avatar in a menu bar and a large one on a profile page.

File: packages/core/src/Avatar.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+Avatar.displayName = 'Avatar'
+
+Avatar.propTypes = {
+  className: PropTypes.string,
+  title: PropTypes.string,
+  subTitle: PropTypes.string,
+  src: PropTypes.string,
+  color: PropTypes.string
```
Comment (unscsprt) [inline packages/core/src/Avatar.js]: It'd be nice if this could display initials if `src` isn't present.

File: packages/core/src/Avatar.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+const StyledImage = styled(Flex)`
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+  background-size: 40px;
+  background-image: url(${props => props.src});
+`
+
+function Avatar({ className, title, subTitle, src, color }) {
```
Comment (unscsprt) [inline packages/core/src/Avatar.js]: Subtitle is one word, so it should be all lowercase.

**Comment by byakimchuk** [comment]
Since this is a new component, do we want to add it to the [public docs](https://github.com/priceline/design-system/tree/master/docs)?

**Comment by James300** [comment]
@byakimchuk we are hopefully going to remove the docs project today and move over to storybook docs.

File: packages/core/src/Avatar.js (hunk 2 above)
Comment (James300) [reply packages/core/src/Avatar.js]: Good suggestion: 
image

Comment (James300) [reply packages/core/src/Avatar.js]: This is the fallback if no src and no initials:
image

File: packages/core/src/Avatar.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import Heading from './Heading'
+import Text from './Text'
+import { User } from 'pcln-icons'
+
+const StyledImage = styled(Flex)`
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  width: ${props => props.size || 40}px;
+  height: ${props => props.size || 40}px;
```
Comment (unscsprt) [inline packages/core/src/Avatar.js]: NB - Want to make `40` the default prop value instead?

**Pull request merged**